resources/istio-init/templates/kyma-job-cleaner.yaml

```diff
@@ -25,10 +25,10 @@ spec:
           - /bin/bash
           - -c
           - |
-            kubectl delete -n istio-system job istio-init-crd-10 --ignore-not-found=true
-            kubectl delete -n istio-system job istio-init-crd-11 --ignore-not-found=true
-            kubectl delete -n istio-system job istio-init-crd-certmanager-10 --ignore-not-found=true
-            kubectl delete -n istio-system job istio-init-crd-certmanager-11 --ignore-not-found=true
+            kubectl delete -n istio-system job istio-init-crd-10-{{ .Values.global.tag | printf "%v" | trunc 32 }} --ignore-not-found=true
+            kubectl delete -n istio-system job istio-init-crd-11-{{ .Values.global.tag | printf "%v" | trunc 32 }} --ignore-not-found=true
```

Comment:
First - that job lacks istio-init-crd-12.
Second - isn't this job useless? Jobs already have a hook `helm.sh/hook-delete-policy: "before-hook-creation"` which will delete the appropriate jobs if such already exist.

There is also one other thing to consider: now in case of Istio version upgrade old jobs will never be removed. Maybe we should instead run this job in post-upgrade step?